/**:
  ros__parameters:
    
    # Signal acquisition
    n_total_channels: 16
    channel_indices_used: [0, 1, 2, 3, 4, 5, 6, 7]
    src: 'plughw:4,0'
    sample_rate: 44100
    hop_size: 44100
    frame_size: 44100 # publish 1 second every second
    microphone_frame_id: 'philbart/base_link'
    codec: 'pcm_s16le'

    # Sound source localization
    speed_sound: 343.
    n_sources: 2
    n_fft: 512
    f_min: 300
    f_max: 8000
    doa_dimension: 3
    array_x_pos: [0.43, 0.43, -0.34, -0.34, 0., 0., 0., 0.]
    array_y_pos: [-0.28, 0.28, 0.30, -0.30, -0.29, 0.29, -0.11, 0.11]
    array_z_pos: [0.395, 0.395, 0.395, 0.395, 0.610, 0.610, 0.660, 0.660]
    ssl_algo: 'MUSIC'

    # Speech activity detection & recognition
    n_silent_frames: 2
    trigger_level: 3.0
    trigger_time: 0.25
    search_time: 0.25
    allowed_gap: 0.25
    pre_trigger_time: 0.25
    min_voice_samples: 31200
    src_match_thresh_rad: .175 # .5 = ~28.6 degrees # .175 rad = ~10 degrees
    lexicon_file: 'config/lexicon.txt'